src/app/SharpRaven/Data/Breadcrumb.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+
+using Newtonsoft.Json;
+
+using SharpRaven.Serialization;
+
+namespace SharpRaven.Data {
+    /// <summary>
+    /// Breadcrumb trail.
+    /// </summary>
+    public class Breadcrumb {
+        private readonly DateTime timestamp;
+
+        public Breadcrumb() 
+        {
+            Category = "log";
```

Comment:
You're right. How I read in documentation and i presume this but I'll change it.

> Each breadcrumb has a few properties of which at least timestamp and category must be provided.